constants/categories.ts

```diff
@@ -4,97 +4,145 @@ const categories: { [route: string]: CopyHolder } = {
   '/food': {
     english: {
       category: 'Food',
+      description:
```

Comment:
again, i'm not certain that this is necessary, as it is repeating what is implied by the title of the section.